src/components/Disputes/DisputeRoundPill.js

```diff
@@ -10,22 +10,29 @@ function DisputeRoundPill({ roundId }) {
 
   const label = `Round ${numberToWord(roundId)}`
   return (
-    <span
+    <div
       css={`
-        padding: 1px ${2 * GU}px;
-        border-radius: 100px;
+        display: flex;
+        align-items: center;
+        justify-content: space-around;
         background: linear-gradient(
           14deg,
           ${theme.accentStart.alpha(0.3)} 0,
           ${theme.accentEnd.alpha(0.3)}88%
         );
-        ${textStyle('label2')}
-        color: ${theme.accent};
-        margin-top: 2px;
+        border-radius: 100px;
+        height: ${2.5 * GU}px;
       `}
     >
-      {label}
-    </span>
+      <span
+        css={`
+          ${textStyle('label2')};
+          color: ${theme.accent};
```

Comment:
Could we also move it 1px down? This label is always displayed in uppercase, so it will make the font look more centered:

```suggestion
          color: ${theme.accent};
          transform: translateY(1px);
```